charts/microgateway/values.yaml

```diff
@@ -12,86 +12,102 @@ image:
+        # config.default.virtualHost.tls.cipherSuite -- Overwrite the default TLS ciphers for frontend connections.
         cipherSuite: ""
 
-  expert:
-    # config.expert.apache -- (string) Expert settings for Apache (multiline string)
+  expert_settings:
+    # config.expert_settings.apache -- Global Apache Expert Settings (multiline string)
+    # @default -- ""
     apache:
-    # config.expert.security_gate -- (string) Expert settings for security gateway (multiline string)
+    # config.expert_settings.security_gate -- Global SecurityGate Expert Settings (multiline string)
+    # @default -- ""
     security_gate:
 
-  # config.env -- List of environment variables (YAML array)
+  # config.env -- List of environment variables. See [Environment variables](#environment-variables)
   env: []
   #  - name: WAF_CFG_EXAMPLE
   #    value: Example_Value
 
-  # config.tlsSecretName -- (string) Name of an existing secret containing the TLS Secrets for the Microgateway.
-  # Virtual Host TLS needs the keys `tls.crt`, `tls.key` and `ca.crt`. Make sure to update `route.tls.destinationCACertificate` accordingly, if used.
-  # Backend TLS needs the keys `backend-client.crt`, `backend-client.key` and `backend-server-validation-ca.crt`.
+  # config.tlsSecretName -- (string) Name of an existing secret containing TLS files.<br>
+  # Virtual Host: Certificate: `tls.crt`, Private key: `tls.key` and CA: `ca.crt`. <br>
+  # :exclamation: Update`route.tls.destinationCACertificate` accordingly.<br>
+  # Backend: Certifate: `backend-client.crt`, Private key: `backend-client.key` and CA: `backend-server-validation-ca.crt`.
+  # @default -- ""
   tlsSecretName:
 
-  # config.passphrase -- (string) Encryption passphrase used for the session.
-  # A random one is generated on each upgrade if not specified here or in `config.existingSecret`
+  # config.passphrase -- Passphrase used for different features (Cookie encryption, URL Encryption, ...)
+  # @default -- * If `passphrase` in `config.existringSecret`<br> * If no passphrase is available, a random is generated.
   passphrase:
   # config.license -- (string) License for the Microgateway (multiline string)
+  # @default -- ""
   license:
-  # config.existingSecret -- (string) An existing secret to be used,
-  # must contain the keys `license` and `passphrase`
+  # config.existingSecret -- (string) Name of an existing secret containing the passphrase or license.<br>
+  # License: `license`<br>
```

Comment:
# license: `license`<br>
When creating the Secret in the examples the key is lowercase.

Reply:
Done